### ISSUE TITLE
Rework Sector Loss Flow

### ISSUE THREAD

**Issue by carlmylo** [moistbois/Moist-Liberation-APR#13]
### Short Summary
We should add a delay and rework the way lost sectors work.

### Description
As of now, when a sector is considered "captured" by the enemy, enemies respawn and repopulate the area as if the sector is activating from the beginning. This looks very janky and is also frustrating to play through.

I think a possible solution would be to not immediately respawn enemies but to perhaps check if it was captured within the last 20 to 30 minutes. If not enough time has passed, it will be impossible to flip the sector back regardless of the current enemy population. If BLUFOR is detected in this recently lost sector, an enemy battlegroup should be dispatched to come defend. Ideally it would spawn both air/paratroopers and a ground battlegroup.

I feel enemies should probably attack in waves as well, which would lead to players having to properly hold the ground for the time. Once 20-30 minutes have passed and all enemies are cleared in the sector, it would then capture the sector. This would create a consequence for failing to defend a sector and would also reward players for being persistent in correcting their mistake by having them defend against waves within a time limit.

TLDR:
1. Lose sector.
2. "Lock" sector ownership for 20-30 minutes to enemies.
3. Send enemy battlegroups within this time, ideally in waves.
4. "Unlock" sector so it's available for capture after 20-30 minutes have passed.

## Challenges
- How the fuck do we pull this off?
- Relevant files for sector are:
    - [Missionframework/scripts/server/sector/attack_in_progress_sector.sqf](https://github.com/moistbois/Moist-Liberation-APR/blob/main/Missionframework/scripts/server/sector/attack_in_progress_sector.sqf) - I think this handles the flow of a sector attack. I _think_ [L48 to L70](https://github.com/moistbois/Moist-Liberation-APR/blob/main/Missionframework/scripts/server/sector/attack_in_progress_sector.sqf#L48-L79) handle the fail condition.
    - [Missionframework/scripts/server/sector/lose_sectors.sqf](https://github.com/moistbois/Moist-Liberation-APR/blob/main/Missionframework/scripts/server/sector/lose_sectors.sqf) - I have no clue how this works but I assume this is called _after_ [attack_in_progress_sector.sqf](https://github.com/moistbois/Moist-Liberation-APR/blob/main/Missionframework/scripts/server/sector/attack_in_progress_sector.sqf)?
    - [Missionframework/scripts/client/remotecall/remote_call_sector.sqf](https://github.com/moistbois/Moist-Liberation-APR/blob/main/Missionframework/scripts/client/remotecall/remote_call_sector.sqf) - Seems to handle the notifications for sector capture status?
    - [Missionframework/functions/fn_getSectorOwnership.sqf](https://github.com/moistbois/Moist-Liberation-APR/blob/main/Missionframework/functions/fn_getSectorOwnership.sqf) - Checks who owns the current sector?
    - All battlegroup related stuff is in the [[battlegroup folder]](https://github.com/moistbois/Moist-Liberation-APR/tree/main/Missionframework/scripts/server/battlegroup)
